diff --git a/config.def.h b/config.def.h
index 58e728e..fb06825 100644
--- a/config.def.h
+++ b/config.def.h
@@ -62,6 +62,9 @@ static const int thumb_sizes[] = { 32, 64, 96, 128, 160 };
 /* thumbnail size at startup, index into thumb_sizes[]: */
 static const int THUMB_SIZE = 3;
 
+/* thumbnail fixed border width */
+static const unsigned int thumb_border_width = 15;
+
 #endif
 #ifdef _MAPPINGS_CONFIG
 
diff --git a/thumbs.c b/thumbs.c
index 37be29f..25fa83e 100644
--- a/thumbs.c
+++ b/thumbs.c
@@ -579,8 +579,7 @@ bool tns_zoom(tns_t *tns, int d)
 	tns->zl = MAX(tns->zl, 0);
 	tns->zl = MIN(tns->zl, ARRLEN(thumb_sizes)-1);
 
-	tns->bw = ((thumb_sizes[tns->zl] - 1) >> 5) + 1;
-	tns->bw = MIN(tns->bw, 4);
+	tns->bw = thumb_border_width;
 	tns->dim = thumb_sizes[tns->zl] + 2 * tns->bw + 6;
 
 	if (tns->zl != oldzl) {
