diff --git a/config.def.h b/config.def.h
index 60eb376..3bd805f 100644
--- a/config.def.h
+++ b/config.def.h
@@ -9,8 +9,8 @@ static char *fontfallbacks[] = {
 #define FONTSZ(x) ((int)(10.0 * powf(1.1288, (x)))) /* x in [0, NUMFONTSCALES-1] */
 
 static const char *colors[] = {
-	"#000000", /* foreground color */
-	"#FFFFFF", /* background color */
+	"#002B36", /* foreground color */
+	"#FDF6E3", /* background color */
 };
 
 static const float linespacing = 1.4;
